cherrelle-morrison/06/exercise.test.js

```diff
@@ -0,0 +1,80 @@
+describe('06', () => {
+
+    // Tip: Use setTimeout to delay the execution of marking
+
+    const listOfPapers = [
+        {
+            subject: "Maths",
+            wasSubmitted: true,
+            markPaper: () => {
+                // add a promise here that resolves after 2 seconds
+                // and print "Maths paper marked"
+
+              return new Promise((resolve,reject) => {
+                setTimeout(() => {
+                    console.log("Maths paper marked");
+                    resolve()
+                }, 2000);                  
+            })
+            }
+        },
+        {
+            subject: "Geology",
+            wasSubmitted: true,
+            markPaper: () => {
+                // add a promise here that resolves after 2 seconds
+                // and print "Geology paper marked"
+                return new Promise((resolve,reject) => {
+                    setTimeout(() => {
+                        console.log("Geology paper marked");
```

Comment:
Ideally, we would have wanted to resolve the text from the promise rather than printing it in the promise, but no marks will be deducted since the instructions could have been misinterpreted.